### DISCUSSION TITLE
Ctrl + Shift + A not working.

### DISCUSSION THREAD

**Discussion by earthmoon** [retorquere/zotero-better-bibtex#2372] [Q&A]
Hello,
With Better Bibtex, I can't use Ctrl + Shift + A to copy. With these keys combination, nothing will be copied but the thing copied before this operation will be.  The Debug ID is: NVYATE3P-apse
Thanks.

**Comment by earthmoon** [comment]
Okay. It works right. In BibLaTeX, there is none of citation style of Ctrl + Shift + A, so there is nothing of Ctrl + Shift + A. But I hope Ctrl + Shift + A will be the Better Bibtex quick copy and Ctrl + Shift + C as the contents of **.bib** file as default. Thanks.

**Reply by retorquere** [reply]
There isn't really an extension point I could hook into. Zotero would have to make it so that a 2nd translator could be configured for quick-copy. Currently you have either a style (reference + bib item) or an export format ("reference"/nothing). If they could do that you could just list BBT there.

**Reply by earthmoon** [reply]
Thanks for your kind reply. As my poor English, I can't understand you well. 

> Currently you have either a style (reference + bib item) or an export format ("reference"/nothing). If they could do that you could just list BBT there.

May you explain it more detailed way?  
Thanks!

**Reply by retorquere** [reply]
When you choose a citation style, you naturally have two things it could output for an item in your library:

1. an in-text citation of the item
2. the entry in the bibliography list for the item

And that's what you get with ctrl-shift-A/C

But if you choose a translator, it can only output one thing for an item in your library.

What you want is that Zotero would change that, so that you can choose for quick-copy either:

1. a citation style, with its two output formats, or
2. not one as they have now, but two export translators, one for ctrl-shift-A and one for ctrol-shift-C.

With that change, you could just select the desired output format for the ctrl-shift-A in Zotero, and it would do what you want. It doesn't seem unreasonable a change if you ask me, but I don't have any influence over their priorities.

I have looked whether I can hook into the quick-copy mechanism to add it myself, but there isn't really a clean entrypoint for it.

**Reply by earthmoon** [reply]
Thanks for your patient reply. You do good. Will you ask Zotero to open that hook for developers?

**Reply by retorquere** [reply]
I can offer support, but you'd have to open the request. It's a functionality you want, I can do the explanation of the technical stuff, but for stuff what you *functionally* want (an option to tie an exporter to ctrl-shift-A) they will want the actual user who wants it to make the request.

**Reply by earthmoon** [reply]
I know now. I will do it right now. Thanks.
